pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -607,14 +624,17 @@ func selectVictimsOnNode(
 		if corev1helpers.PodPriority(p.Pod) < podPriority {
 			potentialVictims = append(potentialVictims, p.Pod)
 			if err := removePod(p.Pod); err != nil {
-				return nil, 0, false
+				finalStatus = framework.NewStatus(framework.Error, err.Error())
+				return nil, 0, finalStatus
```

Comment:
```suggestion
				return nil, 0, framework.NewStatus(framework.Error, err.Error())
```

and the others below

Reply:
Done